earth_enterprise/src/server/wsgi/search/plugin/custom_POI_search_handler.py

```diff
@@ -21,7 +21,7 @@
 import logging.config
 from string import Template
 import urllib2
-import xml.etree.cElementTree as ET
+import defusedxml.cElementTree as ET
```

Comment:
We're going to run into trouble in this file where we call `ET.SubElement` and `ET.tostring`, neither of which exist in defusedxml.